next-i18next-static-site-config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Please rename that file to the `next-i18next-static-site.config.js`. It's a pattern to "end" the config file written in js or ts with `.config` suffix (with dot before).

Reply:
https://github.com/xairoo/next-i18next-static-site/tree/main/examples/web

This is the example I followed, and they are using it without a dot before. I am not sure if that would work with that. Would try manually and update over here.

Reply:
You import that config inside `next.config.js` to whatever you name it, it should work :)

Reply:
Hmm. Got the point. Thanks 😃